pom.xml

```diff
@@ -121,6 +121,16 @@ SOFTWARE.
       <artifactId>files-adapter</artifactId>
       <version>0.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
```

Comment:
@g4s8 As far as I understand, you importing log4j1.2 into the project, but I heard there are some issues with that in Java 9. 
Please take a look at https://blogs.apache.org/logging/entry/moving_on_to_log4j_2

Reply:
@fanifieiev I checked it on JDK13 and it's working fine, you may want to submit a ticket with proposal for update, but I don't think we need to do it right now